docs/metrics.md

```diff
@@ -166,39 +166,49 @@ soroban.host-fn-op.read-entry                | meter     | number of entries rea
+soroban.host-fn-op.read-ledger-byte          | meter     | number of entry (data + code) bytes in read entries
 soroban.host-fn-op.read-data-byte            | meter     | number of data bytes in read entries
 soroban.host-fn-op.read-code-byte            | meter     | number of code bytes in read entries
-soroban.host-fn-op.write-ledger-byte         | meter     | number of data + code bytes in written entries
+soroban.host-fn-op.write-ledger-byte         | meter     | number of entry (data + code) bytes in written entries
 soroban.host-fn-op.write-data-byte           | meter     | number of data bytes in written entries
 soroban.host-fn-op.write-code-byte           | meter     | number of code bytes in written entries
 soroban.host-fn-op.emit-event                | meter     | number of events emitted
 soroban.host-fn-op.emit-event-byte           | meter     | number of event bytes emitted
 soroban.host-fn-op.cpu-insn                  | meter     | metered cpu instructions
 soroban.host-fn-op.mem-byte                  | meter     | metered memory bytes
-soroban.host-fn-op.invoke-time-nsecs         | meter     | time [nsecs] spent in `invoke_host_function`
-soroban.host-fn-op.cpu-insn-excl-vm          | meter     | metered cpu instructions excluding VM instantation
-soroban.host-fn-op.invoke-time-nsecs-excl-vm | meter     | time [nsecs] spent in `invoke_host_function` excluding VM instantation
+soroban.host-fn-op.invoke-time-nsecs         | timer     | time spent in `invoke_host_function`
+soroban.host-fn-op.cpu-insn-excl-vm          | meter     | metered cpu instructions excluding VM instantiation
+soroban.host-fn-op.invoke-time-nsecs-excl-vm | timer     | time spent in `invoke_host_function` excluding VM instantiation
+soroban.host-fn-op.invoke-time-fsecs-cpu-insn-ratio         | histogram | ratio between invoke_time (femto-seconds) and cpu instructions
+soroban.host-fn-op.invoke-time-fsecs-cpu-insn-ratio-excl-vm | histogram | ratio between invoke_time (femto-seconds) and cpu instructions excluding VM instantiation
 soroban.host-fn-op.max-rw-key-byte           | meter     | bytes of the largest key in entries read/written
 soroban.host-fn-op.max-rw-data-byte          | meter     | bytes of the largest data entry read/written
 soroban.host-fn-op.max-rw-code-byte          | meter     | bytes of the largest code entry read/written
 soroban.host-fn-op.max-emit-event-byte       | meter     | bytes of the largest event emitted
-soroban.host-fn-op.success                   | meter     | if `InvokeHostFunctionOp` results in a success
-soroban.host-fn-op.failure                   | meter     | if `InvokeHostFunctionOp` results in a failure
+soroban.host-fn-op.success                   | meter     | number of successful `InvokeHostFunctionOp` operations
+soroban.host-fn-op.count                     | meter     | number of total `InvokeHostFunctionOp` operations
```

Comment:
we don't need this metric (total count): count is already recorded in any other metric that tracks `InvokeHostFunctionOp` data, like `soroban.host-fn-op.invoke-time-nsecs`

Reply:
Ah yes, we didn't have it before (everything before was "meter"). Now we indeed don't need the count. I've reverted that commit (and keeps the "failure" metric).